proto/rpc.proto

```diff
@@ -177,11 +178,14 @@ message ChainId {
 
 // ChainInfo returns chain configuration
 message ChainInfo {
-  ChainId chainid = 1;
-  uint32 bpnumber = 2;
+  ChainId id = 1;
+  uint32 bpNumber = 2;
```

Comment:
why change these two? I think for consistency it would be better to keep them as they were. Otherwise it should also be `maxBlockSize` etc.

Reply:
sorry about that. 
I just want to resolve naming convention inconsistency. but I didn't have to fix it right now.

I know there's still a lot left besides where I've fixed it. We'll discuss it next time and fix it.